### DISCUSSION TITLE
Suggestion to add a screen reader or tts for jak and daxter.

### DISCUSSION THREAD

**Discussion by Computerfan23** [open-goal/jak-project#2878] [Ideas]
So i was suggested to open a separate issue for accessibility features to keep it more organiized. So i will start with this.
Since there is no way for me to read any text in jak and daxter since i am blind, i was wondering if it would be possible to add the support for either screen reader or tts. There are a few libraries that could work.
What i mean with support for text are the menu text and progress numbers, like number of power cells etc.
Hope this was clear enough.

**Comment by Zedb0T** [comment]
>Computerfan23 
Hi!
As a blind gamer, i have run into the problem where i have no idea where a ledge is. I often fall into water most of the time or cannot tell if there is a ledge in front of me that i can jump onto.
I have a suggestion to solve this. Could it be possible to have a sound cue telling you of where ledges are? And a sound cue or warning to tell me i am near a gap or ledge and about to fall off?

**Comment by Computerfan23** [comment]
So what will this be? Is it going to be a place to gather accessibility? This is first time i have seen a discussion on github, so i am not familiar with how it worjs.

**Reply by fabjan** [reply]
Caveat: I speak for myself in the following, and I am trying to describe open source development in general here. I do not know exactly how this project moves forward!

I think this is a great place to have the discussion. I don't know how well GitHub works with a screen reader, but GitHub Discussions let you create threads like this one about TTS and everyone who are interested can see it and participate.

Threads can be upvoted so everyone in the community can see what things have gathered the most interest. They work a little bit like Reddit in that way. Threads are trees of comments in reply to each other (I think so at least, I have not used Discussions much).

I will remind everyone reading that this is an open source project where all developers contribute on their free time and they will work on the thing they most feel like working on. Some developers may want to do fun technically challenging things, other developers might want to do things that they think are fun to make for others in the community (or for themselves).

New developers may start contributing to the project of course, so the number of people programming can increase, but it is not guaranteed that everything anyone programs will get merged into the project, there are many different balances to strike and ultimately it is up to the maintainers of a project to decide what they are OK with taking in (because everything requires maintenance).

**Comment by Computerfan23** [comment]
Ah, i see. I feel it is hard for me to navigate github since it is a lot of tabs and links and a lot of info which i have to scroll past.
So i have no idea how to do threads in a discussion, since i see no easy way for me to do it.

**Reply by fabjan** [reply]
I see. Well, what is needed to try and figure out a complex feature set like accessibility for visually impaired persons for an old console game is going to be a place for persistent discussion which people can be linked to and read the old comments and navigate replies. These kinds of discussions will take many months to have and threads might branch off. There might be some technical discussions alongside any discussion about what features one wants. Discord is likely not a great fit, and apparently GitHub Discussions are hard to navigate.

Do you know any other online tools or forums that are easy to use with visual impairment and would allow you to host a discussion about this topic? Maybe some more old school web forum technology? If not I think this might be the best we have for now. Those of us who are not visually impaired probably have much less experience figuring these things out, and you probably have more knowledge of which discussion platforms are most usable. It seems to me that one would want something that keeps threads organized and navigable without having to read all of the page in a linear fashion every time one comes back to the discussion.
